src/main/java/org/umssdiplo/automationv01/core/managepage/organizationalstructuremenu/CreateDepartmentPage.java

```diff
@@ -0,0 +1,122 @@
+//        CommonEvents.jsClickElement(activeSubmenu);
+//    }
+
+
+
+    public void setDepartmentName(String departmentName) {
+        CommonEvents.isVisible(this.departmentName);
+        CommonEvents.setInputField(this.departmentName, departmentName);
+    }
+
+    public void setParent(String parent) {
+        CommonEvents.isVisible(this.matSelect);
+        CommonEvents.jsClickElement(this.matSelect);
+        WebElement selected = CommonEvents.selectMatOption(this.matSelect, parent);
+        CommonEvents.jsClickElement(selected);
+    }
+
+    public void clickRegistrar() {
+        CommonEvents.isVisible(registrar);
+        CommonEvents.jsClickElement(registrar);
+    }
+
+//    public String getDepartment(String departmentName) {
+//        WebElement sCellValue;
+//        try {
+//            sCellValue = CommonEvents.findWebElementFromMatTable(table, departmentName);
+//            System.out.println("Department: " + sCellValue.getTagName() + " " + sCellValue.getText());
+//            System.out.println(sCellValue.getText().toString());
+//        } catch (Exception e) {
+//            return null;
+//        }
+//        return sCellValue.getText().toString();
+//    }
+
+//    public void actionDepartment(String departmentName, String action) {
+//        WebElement actionButton = CommonEvents.findButtonByCellofMatTable(table, departmentName, action);
+//        System.out.println(actionButton.getTagName() + " " + actionButton.getText());
+//        CommonEvents.jsScrollDown(addDepartment);
+//        CommonEvents.mouseOver(actionButton);
+//        CommonEvents.jsClickElement(actionButton);
+//    }
+
+//    public void editDepartment(String name, String childname, String action) {
+//        setDepartmentName(name);
+//        CommonEvents.isVisible(this.matSelect);
+//        CommonEvents.jsClickElement(this.matSelect);
+//        WebElement selected = CommonEvents.selectMatOption(this.matSelect, childname);
+//        CommonEvents.jsClickElement(selected);
+//        CommonEvents.escape();
+//        CommonEvents.jsClickElement(saveButton);
+//    }
+//
+//    public void editDepartmentChild(String name, String childname, String action) {
+//        setDepartmentName(name);
+//        CommonEvents.isVisible(this.editChildSelect);
+//        CommonEvents.jsClickElement(this.editChildSelect);
+//        WebElement selected = CommonEvents.selectMatOption(this.editChildSelect, childname);
+//        CommonEvents.jsClickElement(selected);
+//        CommonEvents.escape();
+//        CommonEvents.jsClickElement(saveButton);
+//    }
+
+//    public String getChild(String child, String parent) {
+//        WebElement sCellValue;
+//        sCellValue = CommonEvents.findChildFromMatTable(table, child, parent);
+//        System.out.println("Department: " + sCellValue.getTagName() + " " + sCellValue.getText());
+//        System.out.println(sCellValue.toString());
+//        return sCellValue.getText().toString();
+//    }
```

Comment:
Let's remove the commented code. We shouldn't have commented code in last version